### PR TITLE
Hide retry button when status changes to UPLOAD_RETRYING instead of directly in hardcoded scenarios.

### LINKED ISSUE #1455: Retry button remains visible when initiating a retry programmatically
For example, when applying the following mechanism to arrange for endless retries, the Retry button remains visible, even after the upload succeeds.

```javascript
uploader = new qq.azure.FineUploader({
    callbacks: {
        onStatusChange: function (id, oldStatus, newStatus) {
            if (newStatus === qq.status.UPLOAD_FAILED) {
                uploader.retry(id);
```

### PR DESCRIPTION
**Pull request by Korijn**
This PR fixes #1455.

### CODE REVIEW

File: client/js/uploader.api.js [hunk 1]
```diff
@@ -172,7 +172,6 @@
 
                 onRetry: function(fileId) {
                     qq(self._templating.getFileContainer(fileId)).removeClass(self._classes.retryable);
```
Comment (rnicholus) [inline client/js/uploader.api.js]: @Korijn This line should be removed as well. Otherwise, all looks good!

**Comment by Korijn** [comment]
Should be good now?

**Comment by rnicholus** [comment]
Yep, looks good. Were you also able to verify that this works as expected for you?

**Comment by Korijn** [comment]
Yes, I'm already using a temporary build sucessfully in production (we really needed the feature).

**Comment by rnicholus** [comment]
Great. I'll push this out as a hotfix release very soon. It will be 5.3.1.

**Comment by Korijn** [comment]
Awesome, thanks for the help.

**Pull request merged**